### ISSUE TITLE
Request to change Sandringham House to St James House 

### ISSUE THREAD

**Issue by SAbdin93** [rcpch/rcpch-audit-engine#1104]
Hi,

There has been a request to change Sandringham House - Salford (currently under Northern Care Alliance NHS Foundation Trust) to St James House, on the platform as they have moved offices. Here are the details. 

NHS Trust Site - ST JAMES HOUSE (RM341)
National Grouping:
 Y62 NORTH WEST COMMISSIONING REGION
Higher Health Authority:
 QOP NHS GREATER MANCHESTER INTEGRATED CARE BOARD 
Parent:
 RM3 NORTHERN CARE ALLIANCE NHS FOUNDATION TRUST
Address:
 PENDLETON WAY
, SALFORD
, GREATER MANCHESTER
 M6 5FW

If there is anything else you need let me know,

Thank you :) 

Saleema

**Comment by eatyourpeas** [comment]
Thanks @SAbdin93 - sorry as usual I have questions. Do please not let this stop you posting questions.

The organisation Sandringham House, ODS code RM366 currently has Northern Care Alliance NHS Foundation Trust as its parent Trust. The ICB is Greater Manchester and the NHS England Region is North West. It is part of the North West CYP Epilepsy Interest Group.

Are you requesting to:
1. Change the name of the organisation from Sandringham House to St James House
2. Change the ODS code of the organisation from RM366 to RM341

Both organisations still exist according to the ODS portal. If I make this change, Sandringham House will nolonger exist in our database and it will become St James House and all the patients associated with Sandringham will automatically move to St James House. If you still want Sandringham to exist in our database I can recreate it with the same relationships but it will have no patients or staff.

**Comment by SAbdin93** [comment]
@eatyourpeas No worries! I asked the organisation for clarification on the above and got the following response: 

"St James’s House is our administrative base but is not the name of the service. The service name is Salford Care Organisation, which is part of the Northern care Alliance. Sandringham House was our previous base. We do not need Sandringham House to continue to exist and it is right that anything under that name should be reassigned to Salford Care Organisation. That will stop us having to change it again if we move offices again too.

In summary- yes, please can Sandringham House be replaced by Salford Care Organisation."

So I think they are now asking for Sandringham House to be changed to Salford Care Organisation.

Hope this clears things up! Happy to go back to her if not.

**Comment by eatyourpeas** [comment]
No problem @SAbdin93 I can make that change. What about the ODS Code? Does that remain the same or does that move to RM341?

**Comment by SAbdin93** [comment]
The ODS Code will remain the same (RM366), they just want the site name to be changed Salford Care Organisation.

Sorry for the confusion!
